### PR TITLE
Rs 2022 03 17

### PR DESCRIPTION
**Pull request by rschlaikjer**

### CODE REVIEW

File: script/Arcueid/Day 9/09_00_ARC09_2B.txt [hunk 1]
```diff
@@ -56,42 +66,49 @@ Hisui scolds me, and I respond while putting on a forced smile.
...
 -- 俺を抱き上げようと駆け寄ってくるも、懸命な顔のまま固まること約１分。
-Even when she thought about rushing over to hold me up, her face remained distressed and continued to freeze in place for around a minute.
+Next, she rushed over as though to hoist me upright,
+ but then seized up with a strained look on her face for another minute.
 }
 [sha:35c7329780abc9778e73e9e20283f56beb57c5c3]{
 -- Page 41, Offset 21422.
 -- あわてて部屋中を歩き回り、琥珀さんを呼びに行くまでさらに数分。
-As she walked around the room in a fluster, it took another few minutes before she went out to call Kohaku-san.
+She then spent another few minutes pacing anxiously around the room,
```
Comment (Avuxo) [inline script/Arcueid/Day 9/09_00_ARC09_2B.txt:85]: don't think this comma is necessary

File: script/Arcueid/Day 9/09_00_ARC09_2B.txt [hunk 2]
```diff
@@ -101,12 +118,14 @@ If I had collapsed thirty minutes earlier, she might have been late to school as
 [sha:ca26f96f595a0b75989a38acede16f5723e9c97f]{
 -- Page 44, Offset 21428.
 -- おいしいごはんも食べたし、こうしていれば貧血も治まるから、仕事に戻っていいよ」
-I got to eat a delicious meal, and my anemia will get better if I continue to rest like this, so you can get back to work now."
+I've had a delicious meal,
+ and if I keep resting like this my anemia will clear up in no time,
```
Comment (Avuxo) [inline script/Arcueid/Day 9/09_00_ARC09_2B.txt:122]: in no time. It's alright if you

File: script/Arcueid/Day 9/09_00_ARC09_2B.txt [hunk 3]
```diff
@@ -136,10 +156,10 @@ I'll―――
 [sha:c2e44615d839f6abbdd474573b95f1d03e00a9da]{
 -- Page 48, Offset 42661. Choice.
 -- 阿良句博士にする。
- call Dr. Arach.
+ go for Prof. Arach this time.
```
Comment (Avuxo) [inline script/Arcueid/Day 9/09_00_ARC09_2B.txt:159]: no shiki don't do it!

File: script/Arcueid/Day 9/09_00_ARC09_2B.txt [hunk 4]
```diff
@@ -136,10 +156,10 @@ I'll―――
...
 }
 [sha:67231cd548176cbb5a7cc91838f85e999f7c7d11]{
 -- Page 48, Offset 42662. Choice.
 -- やっぱり琥珀さんでいい。
- Actually, Kohaku-san is fine.
+ be just fine with Kohaku-san, thanks.
```
Comment (Avuxo) [inline script/Arcueid/Day 9/09_00_ARC09_2B.txt:164]: revert (though i do like that you made it fit with I'll---)

**Review by Avuxo — APPROVED** [review]

File: script/Arcueid/Day 9/09_00_ARC09_2B.txt (hunk 4 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 9/09_00_ARC09_2B.txt:164]: How about 
> ...To be honest, I don't need a medical exam, but I don't think Hisui will leave unless I agree.
> If I have to, then―――
>   1. I'll have Prof. Arach take a look.
>   2. I think I'd much rather Kohaku-san do it.

Comment (Avuxo) [reply script/Arcueid/Day 9/09_00_ARC09_2B.txt:164]: I still think thats quite strong for やっぱり琥珀さんでいい。but that's better

**Pull request merged**